test/Unit/Api/Resources/CurrencyTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+namespace Bigcommerce\Test\Unit\Api\Resources;
+
+use Bigcommerce\Api\Resources\Currency;
+use Bigcommerce\Api\Client;
+
+class CurrencyTest extends ResourceTestBase
+{
+    public function testCreatePassesThroughToConnection()
+    {
+        $currency = new Currency((object)array('id' => 1));
+        $this->connection->expects($this->once())
+            ->method('post')
+            ->with($this->basePath . '/currencies', (object)array('id' => 1));
+
+        $currency->create();
+    }
+
+    public function testUpdatePassesThroughToConnection()
+    {
+        $product = new Currency((object)array('id' => 1));
```

Comment:
I'd change these bits to be `$currency` instead of `$product` in these last 2 tests